<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ewfRelat</class>
 <widget class="QDialog" name="ewfRelat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relatório</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_case">
                <property name="styleSheet">
                 <string notr="true"> font: bold 8pt &quot;DroidSans&quot;;</string>
                </property>
                <property name="text">
                 <string>Caso Nº</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_caseNumber">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>23</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true"> font: bold 8pt &quot;DroidSans&quot;;</string>
                </property>
                <property name="text">
                 <string>Evidência Nº</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_evidenceNumber">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>23</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_Examiner">
                <property name="minimumSize">
                 <size>
                  <width>200</width>
                  <height>23</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true"> font: bold 8pt &quot;DroidSans&quot;;</string>
                </property>
                <property name="text">
                 <string>Nome do Examinador</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="2" column="0">
             <widget class="QTextEdit" name="textEdit_notes">
              <property name="minimumSize">
               <size>
                <width>470</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>470</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true"> font: bold 8pt &quot;DroidSans&quot;;</string>
              </property>
              <property name="text">
               <string>Notas:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true"> font: bold 8pt &quot;DroidSans&quot;;</string>
                </property>
                <property name="text">
                 <string>Descrição</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTextEdit" name="textEdit_description">
                <property name="minimumSize">
                 <size>
                  <width>470</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>470</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ok">
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cancel">
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
